pkg/executor/task_handle.go

```diff
@@ -77,3 +77,15 @@ func StopAndEraseOutput(handle TaskHandle) (errorCollection errcollection.ErrorC
 
 	return
 }
+
+// GetWaitChannel returns channel that will return result (any encountered error) of
+// Wait() method in provided handle.
+func GetWaitChannel(handle TaskControl) <-chan error {
```

Comment:
@skonefal I don't think that we need to export this function. It would make sens to rename it to `waitForTask()` (is it seems to me that waiting is the most important part of the fuction) and place it where `getWaitTimeoutChan()` can be found.